src/api/entityResolution/index.ts

```diff
@@ -273,4 +274,25 @@ export class EntityResolutionAPI extends Request {
       params: params
     });
   }
+
+  /**
+   * Get entity resolution server status
+   */
+  getEntityResolutionServerStatus(
+    this: Request<{status: string}>,
```

Comment:
we can use the type `EntityResolutionServerStatus` instead of string

```suggestion
    this: Request<{status: EntityResolutionServerStatus}>,
```